packages/config/src/normalize/events.js

```diff
@@ -3,14 +3,17 @@ const EVENTS = [
    * `onInit` - Runs before anything else
    */
   'onInit',
-  // `onGetCache` - Fetch previous build cache. Not implemented yet.
-  'onPreGetCache',
-  'onGetCache',
-  'onPostGetCache',
-  // `onInstall` - Install project dependencies. Not implemented yet.
-  'onPreInstall',
-  'onInstall',
-  'onPostInstall',
+
```

Comment:
[pebble] [Non blocking] - What do you think about removing them all together instead of commenting them out. Commented code might lose context over time. Removing it will force us to rethink those if we introduce them again.
Regardless most of them are already under `LEGACY_EVENTS` for reference.

Reply:
Yes, you're right. Done in 48a8ea3cd8bdbb596e17e4c844ac811fb6ed9f92